### ISSUE TITLE
Quick Question: How does the model train?

### ISSUE THREAD

**Issue by RamonYeung** [lajanugen/zeshel#1]
Dear Lajanugen,

I notice your paper as soon as the accepted paper list of ACL'19 is available. What a novel approach and great work!

I have two questions about the training procedure.
Q1: What do you mean by "the model is trained using a softmax loss"? Is it referring to the Cross Entropy Loss?
Q2: Do you concat the unseen classes with seen classes during training? (but no samples for unseen classes available).

Thanks and best wishes for you,
Ramon

**Comment by RamonYeung** [comment]
Closing since code release. Thank you Lajanugen!
